sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -797,6 +797,10 @@ public static void addTransactionBatchResultInResponse(
     }
 
     @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static int getPayloadLength(TransactionalBatchResponse transactionalBatchResponse) {
+        return transactionalBatchResponse.getResponseLength();
+    }
+  
     public static ThroughputControlMode getThroughputControlMode(ThroughputControlGroup throughputControlGroup) {
```

Comment:
Did we end up removing the warning on this by mistake?

Reply:
yes I will add that. its a mistake.